app/views/handlers/confirmation_email.py

```diff
@@ -26,12 +25,13 @@ def __init__(
         self,
         session_store: SessionStore,
         schema: QuestionnaireSchema,
-        page_title: Optional[str] = None,
-        serialised_email: Optional[str] = None,
+        page_title: str | None = None,
+        serialised_email: str | None = None,
     ):
         if not self.is_enabled(schema):
             raise ConfirmationEmailNotEnabled
 
+        # type ignore: session_data is populated at login therefore won't be None for confirmation email
```

Comment:
Minor, use caps `# Type ignore:` as per doc and for consistency. Applies to all.